base-node/index.d.ts

```diff
@@ -197,6 +199,19 @@ export class BaseNode<M extends Meta = Meta> {
    */
   remoteSubprotocol: string | undefined
 
+
+  /**
+   * Headers set by remote node.
+   * By default, it is an empty object.
+   * 
+   * ```js
+   * if (Object.keys(node.remoteHeaders).length > 0) {
```

Comment:
A good example shows why you should use some API.

For this method, we can show an example with different string in the action according to user’s locale.

Reply:
I've update the example. But I'm not sure if it is what you meant?